config/config.yaml

```diff
@@ -161,6 +161,15 @@ storage:
   # More info: https://qdrant.tech/documentation/guides/distributed_deployment/#shard-transfer-method
   shard_transfer_method: null
 
+  # Default parameters for vectors.
+  vectors_config:
+    # Whether vectors should be stored in memory or on disk.
+    on_disk: false
```

Comment:
Thought it would effectively be the same, we might want to prefer `null` as default here to not set it at all.

Wdyt? @generall

Reply:
+